### ISSUE TITLE
Installation/Test Issue

### ISSUE THREAD

**Issue by franciscozorrilla** [Ecogenomics/CheckM#185]
Hello, I am trying to install and test checkM but I seem to be having some problems.

I have looked through all the issues in the repository, and this error message/issue seems to be unreported as far as I can tell.

I followed the instructions in the wiki, making sure all dependencies were present, and tried both `pip install` and `conda install`. Below is the list of installed packages in my checkM conda environment.

![image](https://user-images.githubusercontent.com/35606471/51133337-0784fc80-1835-11e9-90db-000faf39037c.png)

In both cases I end up with the same error message when I try running the following test:

`checkm test TESTDIR`

![image](https://user-images.githubusercontent.com/35606471/51133613-c3462c00-1835-11e9-9759-f1bf248e735f.png)

After running this, I get my output folder TESTDIR, with an empty folder called results.

I also made sure to download and set the data directory as demonstrated [here](https://hackmd.io/cMQgeGF6TOuuLJvCjQyo0Q#).

I am not sure how to interpret the error message, but I do not believe this to be a memory associated issue. I currently have a job queued up on my institution's HPC cluster, with more than enough RAM (64 GB RAM node), to determine if this is a memory issue. Besides this, any clue as to what could be going wrong?

Best wishes,
Francisco

**Comment by dparks1134** [comment]
Hello Francisco. Thank you for bringing this to my attention. This is a bug with CheckM v1.0.13 and will be fixed in the next release. In the meantime, I suggest just running a test genome through the lineage_wf command to verify the installation is correct. The "test" command doesn't do anything beyond verifying proper installation.

**Comment by franciscozorrilla** [comment]
Hello Donovan, thank you for your speedy response!

I tried running lineage_wf as shown below

 `checkm lineage_wf test_data/ TESTDIR`

![image](https://user-images.githubusercontent.com/35606471/51136593-2a1b1380-183d-11e9-902e-e502a722db16.png)

followed by

![image](https://user-images.githubusercontent.com/35606471/51136641-4d45c300-183d-11e9-9859-22e60ef99242.png)

I tried searching for this issue in the repository, but it also seems to be unreported.
It seems like there is some issue with a missing/old library or filepath.

I tried checking the $PATH variable as shown below

```
echo $PATH
/c3se/users/zorrilla/Hebbe/.conda/envs/checkm_env/bin:.......
```

And this folder seems to contain hmmsearch and hmmfetch.

![image](https://user-images.githubusercontent.com/35606471/51136997-19b76880-183e-11e9-9a7a-74f1d54232fb.png)

Any clue how I can solve this?

Best wishes,
Francisco
